specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/azureFirewall.json

```diff
@@ -739,21 +739,6 @@
             "modelAsString":true
          }
       },
-      "ProvisioningState":{
```

Comment:
Looks like breaking change. not acceptable for already published version. please revert and save for the next monthly branch

Reply:
No, this is a duplicate